src/anyio/_core/_testing.py

```diff
@@ -63,10 +67,10 @@ def get_running_tasks() -> list[TaskInfo]:
     """
     Return a list of running tasks in the current event loop.
 
-    :return: a list of task info objects
+    :return: a sequence of task info objects
```

Comment:
That was my first thought too, but:

1. it is factually returning a list
2. a Sequence (or even a MutableSequence) would narrow the return type, thus breaking backwards compatibility, albeit only from the type checker standpoint.

I see no ideal solution here, so I chose the least disruptive way forward. Thoughts?

Reply:
Let's just say "list" instead of "sequence" here in the docstring.

As a broader comment, I tend to treat type annotations as documentation rather than a stable API, and thus don't mind updating them in minor versions if that otherwise makes sense.  But continuing to use `list` here seems best anyway.

Reply:
I've reverted the wording.